### PR TITLE
[Catalog] Fixed some swift Imports

### PR DESCRIPTION
**Pull request by randallli**
Changed some imports to be at the component level instead of all of material components.

### CODE REVIEW

File: catalog/MDCCatalog/AppDelegate.swift [hunk 1]
```diff
@@ -17,7 +17,21 @@ limitations under the License.
 import UIKit
 
 import CatalogByConvention
-import MaterialComponents
+
```
Comment (willlarche) [inline catalog/MDCCatalog/AppDelegate.swift:20]: Why do we prefer 14 imports to 1?

Comment (randallli) [reply catalog/MDCCatalog/AppDelegate.swift:20]: Internally we do not have a global target that includes everything MDC.

Comment (willlarche) [reply catalog/MDCCatalog/AppDelegate.swift:20]: Why is this an issue now?

Comment (randallli) [reply catalog/MDCCatalog/AppDelegate.swift:20]: This is to get our catalog working on internal build.

**Review by jverkoey — APPROVED** [review]

**Pull request merged**